### DISCUSSION TITLE
热力图配置类型问题

### DISCUSSION THREAD

**Discussion by MangMax** [zouyaoji/vue-cesium#538] [Q&A] [answered]
![image](https://github.com/zouyaoji/vue-cesium/assets/68525017/34d3cf98-3de4-4d37-b3c8-040a6a394229)
![image](https://github.com/zouyaoji/vue-cesium/assets/68525017/02eeebda-4b6f-42d6-9030-e35d6ce77d55)
![image](https://github.com/zouyaoji/vue-cesium/assets/68525017/3c902320-6ee3-4fe8-8ead-07f651a0cfec)

源码中热力图配置存在多个必填项，根据文档的描述实测不填是不影响的，但是类型上会报错

**Comment by zouyaoji** [comment, marked as answer]
确实哈。 你先本地改成这样，下个版本解决。
```
type HeatmapConfiguration = Partial<h337.HeatmapConfiguration> & {
  minCanvasSize?: number
  maxCanvasSize?: number
  radiusFactor?: number
  spacingFactor?: number
}
```

**Reply by MangMax** [reply]
ok,感谢回复
